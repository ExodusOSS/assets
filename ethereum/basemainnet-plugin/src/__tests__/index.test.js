import { runEvmIndexTestSuite } from '@exodus/ethereum-api/src/__tests__/index.testsuite.js'

import assetPlugin from '../index.js'

describe(`basemainnet index.js test`, () => {
  runEvmIndexTestSuite({ assetPlugin })
})
