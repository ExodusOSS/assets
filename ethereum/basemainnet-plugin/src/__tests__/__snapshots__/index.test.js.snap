exports[`basemainnet index.js test evm fee data snapshot 1`] = `
{
  "baseFeePerGas": "0.2 Gwei",
  "eip1559Enabled": true,
  "fuelThreshold": "750000 Gwei",
  "gasPrice": "0.2 Gwei",
  "gasPriceMaximumRate": 1.3,
  "gasPriceMinimumRate": 0.6,
  "gasPriceMultiplier": 1,
  "max": "1 Gwei",
  "min": "0.001 Gwei",
  "origin": "0.2 Gwei",
  "tipGasPrice": "0.02 Gwei",
}
`;
